src/chrome/content/rules/Duke.edu.xml

```diff
@@ -0,0 +1,147 @@
+			 - bigdata.duke.edu
+			 - www.fhi.duke.edu
+			 - lemur.duke.edu
+
+	Non-functional host in *.dukeexechealth.org
+		SSL peer certificate was not OK:
+			 - dukeexechealth.org
+			 - www.dukeexechealth.org
+	
+	Non-functional host in *.dukeexecutivehealth.org:
+		SSL peer certificate was not OK:
+			 - dukeexecutivehealth.org
+			 - www.dukeexecutivehealth.org
+		
+-->
+<ruleset name="Duke University (partial)">
+	<!-- *.duke.edu -->
+	<target host="duke.edu" />
+	<target host="www.duke.edu" />
+	<target host="aahvs.duke.edu" />
+	<target host="academiccouncil.duke.edu" />
+	<target host="arts.duke.edu" />
+	<target host="bassconnections.duke.edu" />
+	<target host="cit.duke.edu" />
+	<target host="civic.duke.edu" />
+	<target host="danceprogram.duke.edu" />
+	<target host="dibs.duke.edu" />
+	<target host="www.dibs.duke.edu" />
+	<target host="divinity.duke.edu" />
+	<target host="dukeforward.duke.edu" />
+	<target host="dukeperformances.duke.edu" />
+	<target host="durham.duke.edu" />
+	<target host="energy.duke.edu" />
+	<target host="english.duke.edu" />
+	<target host="entrepreneurship.duke.edu" />
+	<target host="financialaid.duke.edu" />
+	<target host="gardens.duke.edu" />
+	<target host="genome.duke.edu" />
+	<target host="giving.duke.edu" />
+	<target host="global.duke.edu" />
+	<target host="globaled.duke.edu" />
+	<target host="globalhealth.duke.edu" />
+	<target host="gradschool.duke.edu" />
+	<target host="hr.duke.edu" />
+	<target host="www.hr.duke.edu" />
+	<target host="humanitieswritlarge.duke.edu" />
+	<target host="inclusive.duke.edu" />
+	<target host="law.duke.edu" />
+	<target host="library.duke.edu" />
+	<target host="math.duke.edu" />
+	<target host="www.math.duke.edu" />
+	<target host="mclibrary.duke.edu" />
+	<target host="mfaeda.duke.edu" />
+	<target host="music.duke.edu" />
+	<target host="my.duke.edu" />
+	<target host="www.nasher.duke.edu" />
+	<target host="newsoffice.duke.edu" />
+	<target host="nicholas.duke.edu" />
+	<target host="www.nicholas.duke.edu" />
+	<target host="nursing.duke.edu" />
+	<target host="olv.duke.edu" />
+	<target host="pratt.duke.edu" />
+	<target host="it.pratt.duke.edu" />
+	<target host="www.pratt.duke.edu" />
+	<target host="provost.duke.edu" />
+	<target host="registrar.duke.edu" />
+	<target host="research.duke.edu" />
+	<target host="sanford.duke.edu" />
+	<target host="graduate.sanford.duke.edu" />
+	<target host="scienceandsociety.duke.edu" />
+	<target host="sites.duke.edu" />
+	<target host="socialmedia.duke.edu" />
+	<target host="stories.duke.edu" />
+	<target host="studentaffairs.duke.edu" />
+	<target host="styleguide.duke.edu" />
+	<target host="theaterstudies.duke.edu" />
+	<target host="today.duke.edu" />
+	<target host="www.today.duke.edu" />
+	<target host="travel.duke.edu" />
+	<target host="trinity.duke.edu" />
+	<target host="undergrad.duke.edu" />
+	<target host="undergraduateresearch.duke.edu" />
+	<target host="wiki.duke.edu" />
```

Comment:
I won't annoy you, but I just stumbled over `lists.duke.edu` and `oie.duke.edu`, which are not yet in the list of targets. :)